assisted-events-scrape/events_scrape/events_scrape.py

```diff
@@ -1,35 +1,29 @@
 #!/usr/bin/env python3
 
-import re
 import os
 import sys
 import time
-import json
-import random
 import urllib3
 import logging
-import hashlib
-import tempfile
 import elasticsearch
 import sentry_sdk
-from datetime import datetime
-from datetime import timedelta
 
-from sentry_sdk import capture_exception
-
-from assisted_service_client import rest
-from contextlib import suppress
+from queue import Queue
 
 from .assisted_service_api import ClientFactory
-from . import process
-from .logger import log
+from utils.logger import log
+from repositories.cluster_repository import ClusterRepository
+from repositories.event_repository import EventRepository
```

Comment:
Add all the Repository objects to the __init__ file 


```suggestion
from repositories import ClusterRepository, EventRepository
```